packages/superset-ui-legacy-preset-chart-big-number/src/BigNumber/BigNumber.jsx

```diff
@@ -86,8 +89,11 @@ const propTypes = {
   renderTooltip: PropTypes.func,
 };
 const defaultProps = {
+  alignTimeRange: false,
```

Comment:
Is `alignTimeRange` required?
If both `fromDateTime` and `toDateTime` are set, would that be enough to apply the fixed domain?

What do you think if we merge `fromDateTime` and `toDateTime` fields to a single field `domain: number[]`?

Reply:
To keep backward capability, I plan to make `alignTimeRange` disabled by default, there will be a new control in Superset that enables this behavior.

`to_dttm` and `from_dttm` (the parsed date range) is not currently available in the backend. I plan to add them as a generic API for all future charts. To keep things simple, I decided not to let the backend handle whether or not to output `to_dttm` and `from_dttm` based on chart types / control options, nor add another layer of data processing in the frontend to do so. `transformProps` should serve that purpose.

Not having `from_dttm` and `to_dttm` in an `domain` array is to follow the same coding style in the backend:
https://github.com/apache/incubator-superset/blob/53ed851339e8607f4609762664d389d60d1ad26e/superset/viz.py#L324-L352

We could have used an array variable `parsed_range` or something, but that adds more API for developers to remember and would make searching the code harder.

Reply:
Got it on the date time. 

I wish there is a clearer naming for `alignTimeRange`. `align` for me means adjusting the time of each data point. Perhaps `fixTimeRange`, `useFixedTimeRange`?